crates/sui-protocol-config/src/lib.rs

```diff
@@ -2226,6 +2232,8 @@ impl ProtocolConfig {
 
             max_txn_cost_overage_per_object_in_commit: None,
 
+            allowed_txn_cost_overage_burst_per_object_in_commit: None,
```

Comment:
Is it expect that we don't set it yet in a protocol config?

Reply:
Yes, as a matter of principle I try to configure protocol config in a separate PR from the one that adds the feature